dna/models.py

```diff
@@ -15,7 +15,6 @@
 from dna.kND import KNearestDatasets
 from dna import utils
 
-
```

Comment:
I like having two newlines between different sections of codes at the top level.

Reply:
I actually do too. That was my mistake.